### DISCUSSION TITLE
remember view type

### DISCUSSION THREAD

**Discussion by unclejef** [stashapp/stash#2445] [Q&A] [answered]
Hi 
is there a way to set a different view type for each page?
e.g. I'd like to set grid view for scenes but keep list view in the tags page
thx!

**Comment by BrikoX** [comment, marked as answer]
You can save custom filters and set the one you prefer as default. 
![Screenshot_146](https://user-images.githubusercontent.com/69597682/160656969-5194a89a-ab61-435b-bac0-0ab933f6fc57.png)

**Reply by unclejef** [reply]
now that's embarrassing I didn't notice the save icon
thank you for pointing out! :)
